CHANGELOG.md

```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.0] Ganymed - 2021-07-09 [:warning:]
```

Comment:
The 0.7.0 link is dead.  
You can add the links in the bottom, should be something like:  
`https://github.com/hyperledger-labs/go-perun/compare/v0.6.0...v0.7.0`.

Reply:
yes, please adjust the bottom list of links